cli/template/base/package.json

```diff
@@ -22,8 +22,8 @@
     "@types/node": "^20.11.20",
     "@types/react": "^18.2.57",
     "@types/react-dom": "^18.2.19",
-    "@typescript-eslint/eslint-plugin": "^7.1.1",
-    "@typescript-eslint/parser": "^7.1.1",
+    "@typescript-eslint/eslint-plugin": "8.0.0-alpha.39",
+    "@typescript-eslint/parser": "8.0.0-alpha.39",
```

Comment:
this is user-facing so we'll hold off until it's stable I tihnk

Reply:
Oh right I'd meant to come back to this, sorry 😅 doing now!